### DISCUSSION TITLE
include group in another group

### DISCUSSION THREAD

**Discussion by deronnax** [pdm-project/pdm#2590] [Q&A] [answered]
Hello.
I would like to have a `test` group, that I would install only in CI, and include it in the general `dev` development group, so developers, by doing `pdm sync --dev`, would have both test dependencies and "normal" dev dependencies like flake8. It would theorically look like this:
```toml
[tool.pdm.dev-dependencies]
test = [
    'pytest', 
    'pytest-timeout'
]
dev = [
    '*test',
    'flake8',
    'ipython',
]
```
I did not find how to do it in the documentation. Can this be done? How?
Thanks

**Comment by frostming** [comment, marked as answer]
> Can this be done?

Partially, only if the referred group is an **optional** dependency group and the project itself is a distribution, then refer it as `mypackage[test]`

Otherwise let's keep an eye on PEP 735

**Reply by deronnax** [reply]
Ah, the project is not a distribution. But thank you.
